docker/environment.yml

```diff
@@ -6,27 +6,39 @@ channels:
   - bokeh
   - plotly  # for jupyter-dash
   - defaults
+
 dependencies:
+
+  # Do not put xclim and ravenpy direct dependencies here to let xclim and ravenpy
+  # manage their own dependencies pinning.
+  #
+  # xclim direct dependencies: https://github.com/conda-forge/xclim-feedstock/blob/master/recipe/meta.yaml
+  # ravenpy direct dependencies: https://github.com/conda-forge/ravenpy-feedstock/blob/master/recipe/meta.yaml
+
+  # Pin latest xclim and ravenpy to avoid downgrading during the second install
+  # phase.  Mamba is quicker to solve dependencies than conda but it is less
+  # precise so accidental downgrade happends.
+  - xclim >= 0.32.1
+  - ravenpy >= 0.7.7
```

Comment:
```suggestion
  - ravenpy >= 0.7.8
```

Reply:
That's funny, you read my mind, I just did this a few minutes ago in https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/95/commits/c9d2ef60d5cdd6b391c004a056fe0c2f9ee2bdce